CFC_DataCollector/recommender/augment_trips_pipeline.py

```diff
@@ -0,0 +1,26 @@
+import get_trips
+#import Profiles
+
+def augment_trips(trip_list):
```

Comment:
AlternateTripGenerating team will do this i.e. provide interface to return such trips.